### PR TITLE
perf: change profile api

### PR DESCRIPTION
**Pull request by fit2bot**
perf: change profile api

### CODE REVIEW

File: src/api/users.js [hunk 1]
```diff
@@ -50,6 +61,7 @@ export function editUserGroup(data) {
     data: data
   })
 }
+
 export function updateUserGroup(id, data) {
   return request({
     url: '/api/v1/users/groups/' + id + '/',
```
Comment (BaiJiangJie) [inline src/api/users.js:67]: The code looks fine and doesn't require any modifications at this time. However, it appears you've made some changes to the API endpoints (`login`, `getProfile`, `getUserList`). The code is written using JavaScript, which could be used in various platforms like Node.js.

To enhance the code's performance for large scale deployment (like backend web applications), consider:

- **Code Refactoring**: Use modern tools such as Linters(e.g. ESLint) to improve readability and follow best practices

- **Performance Optimization**:
  - **Preventing SQL Injection**: Make sure that no query parameter contains user-supplied values
  - Using efficient caching strategies if required
  - Reducing number of HTTP requests where possible
  
- **Security Measures**: Implement input validation, sanitization techniques, use HTTPS, properly validate tokens etc.

In general, ensure that the structure of each component functions independently, without side effects from one calling another. Also remember to include unit tests to confirm all edge case scenarios work correctly.

**Review by ibuler — APPROVED** [review]
